eclair-core/src/main/scala/fr/acinq/eclair/db/DbEventHandler.scala

```diff
@@ -119,8 +118,14 @@ class DbEventHandler(nodeParams: NodeParams) extends Actor with ActorLogging {
       channelsDb.updateChannelMeta(e.channelId, event)
 
     case u: LocalChannelUpdate =>
-      if (Announcements.isEnabled(u.channelUpdate.channelFlags)) {
-        auditDb.addChannelUpdate(u)
+      u.previousChannelUpdate match {
+        case Some(previous) if
+          u.channelUpdate.feeBaseMsat == previous.feeBaseMsat &&
+          u.channelUpdate.feeProportionalMillionths == previous.feeProportionalMillionths &&
+          u.channelUpdate.cltvExpiryDelta == previous.cltvExpiryDelta &&
+          u.channelUpdate.htlcMinimumMsat == previous.htlcMinimumMsat &&
+          u.channelUpdate.htlcMaximumMsat == previous.htlcMaximumMsat => ()
+        case _ => auditDb.addChannelUpdate(u)
```

Comment:
Have you seen `Announcements.areSame`? It's not exactly what you need here (flags should be ignored) though.

Reply:
Yes I've seen it but it didn't match exactly what I needed and I thought it would be better to explicitly list the fields we want to compare.